### ISSUE TITLE
[Question] Does the jailbreak tweak need to be reinstalled after every reboot on semitethered/semiuntethered jailbreaks?

### ISSUE THREAD

**Issue by CSab6482** [open-imcore/imessage-rest#24]
Every time I turn my phone back on and rejailbreak it with checkra1n, the tweak stays stuck at "Server is starting" with the yellow indicator light, and I can't get it to work until I uninstall the tweak and run the `dpkg -i` again. Is this a coincidence, or does anybody else face a similar issue?

**Comment by CSab6482** [comment]
I can get the server running on iOS, my question is about the effect that a semi untethered jailbreak has on the ability to run the server. Every time I reboot and need to rejailbreak, I also need to uninstall and reinstall Barcelona to get it running.

**Comment by CarlAmbroselli** [comment]
I don't think you need to reinstall Barcelona, you just need to run it in i.e. a tmux session.

Given that when it's semi-untethered you need do manual actions to jailbreak it anyhow, I don't think it's so difficult to also run that single SSH command to start up the server?

But if it's really bugging you, then why not start it using activator?

**Comment by CSab6482** [comment]
What is the SSH command? I'll give it a few test runs to see how it goes.

Additionally, I do still consider this something that should be looked at, especially if Barcelona is ever publicly released as a full tweak. The GUI should be functional for all use cases/users when jailbroken, unless the tweak explicitly states that sometimes it can only run via CLI.

It's up to Eric how he decides to implement it though.

**Comment by CarlAmbroselli** [comment]
Maybe I'm missing something, but there is a UI? I always just ran `./mautrix-imessage` from the folder where that executable + the Barcelona executable + config.yaml is stored.

**Comment by CSab6482** [comment]
Ah okay, we're on totally different pages. I haven't used that updated version that you are running. I am using a tweak that comes with its own app.

https://user-images.githubusercontent.com/53708281/154964009-aaea4db3-f00d-4e0b-b62a-8c950270a774.MP4

I first started using this tweak around one year ago, but I haven't been able to download an updated version for about 10 months since I don't know how to compile it. I was just downloading prebuilt versions from [here](https://github.com/open-imcore/barcelona/actions).

**Comment by CarlAmbroselli** [comment]
Actually, currently you also need https://github.com/mautrix/imessage/pull/44 to be merged, https://github.com/open-imcore/barcelona/pull/36 to include in your steps and https://github.com/open-imcore/barcelona/pull/40 as well.

**Comment by CSab6482** [comment]
Once I do have Barcelona running with these updated instructions, what frontend do I use for it?

**Comment by CarlAmbroselli** [comment]
This will just bridge your messages into matrix, so any matrix client can be used (Beeper, Element, FluffyChat, Neo, ...)
